### DISCUSSION TITLE
recommend https on all the pages

### DISCUSSION THREAD

**Discussion by nezda** [lum-ai/odinson#293] [Ideas]
You should consider enforcing `https` on all the pages: I think that setting would be https://github.com/lum-ai/odinson/settings/pages ? (https://github.com/nezda/yawni.github.io/settings/pages on my project). 

![image](https://user-images.githubusercontent.com/179091/120446555-49a03180-c34f-11eb-91be-e4fe5351f1c5.png)

Why SSL everywhere?

- Really, why not? https://letsencrypt.com 
- Better search engine ranking
- Improves customer trust
- Google made SSL mandatory in 2018
    - punishment: ‘Not Secure’ on the URL bar

Kind regards,
- Luke

**Comment by BeckySharp** [comment]
@kwalcock is this something you can look at/consider for us? or should we be prodding someone else?

**Comment by kwalcock** [comment]
If I understand it correctly, https is available for all pages; it's just not enforced on some.  On github pages (github.com/lum-ai), http requests are automatically converted to https, and that might be good because we don't want people inserting a virus into the code during transmission.  On gh pages (gh.lum.ai) mostly providing documentation, requests are not automatically converted, but are still available.  I would counter with why not give people a choice?  Encryption is not free (e.g., https://dl.acm.org/doi/10.1145/2674005.2674991).  Someone who is paranoid that our documentation (or maybe images) may be manipulated can use it; someone else who wants to save all that energy and put it into bitcoin mining instead, might skip it.

There is indeed just a checkbox for it and it seems like we already have the certificate.  If someone feels strongly, please go ahead.

**Reply by nezda** [reply]
Sounds like you don't like padlocks 🔒 next to your urls @kwalcock : to each their own 👍.
